api/users/serializers.py

```diff
@@ -298,3 +298,13 @@ class UserAccountDeactivateSerializer(BaseAPISerializer):
 
     class Meta:
         type_ = 'user-account-deactivate-form'
+
+
+class UserChangePasswordSerializer(BaseAPISerializer):
```

Comment:
Is there a reason you subclassed `BaseAPISerializer` instead of `JSONAPISerializer`?

Reply:
It doesn't seem to make any appreciable difference in this case and is more in-line with similar user settings views/serializers `UserAccountExport` and `UserAccountDeactivate`